packages/uikit-react-native-foundation/src/components/TextInput/index.tsx

```diff
@@ -15,6 +15,10 @@ const TextInput = React.forwardRef<RNTextInput, Props>(function TextInput(
   const variantStyle = colors['ui']['input'][variant];
   const inputStyle = editable ? variantStyle.active : variantStyle.disabled;
   const underlineStyle = variant === 'underline' && { borderBottomWidth: 2, borderBottomColor: inputStyle.highlight };
+  const fontStyle = {
+    ...typography.body3,
+    lineHeight: typography.body3.fontSize ? typography.body3.fontSize * 1.2 : undefined,
```

Comment:
While vertical alignment is supported on Android, it is not supported on iOS.
For this reason, if the lineHeight is not adjusted proportionally based on fontSize in TextInput, there may be an issue with misalignment